app/src/main/java/org/oppia/app/profile/AdminAuthViewModel.kt

```diff
@@ -15,7 +15,6 @@ class AdminAuthViewModel @Inject constructor() : ObservableViewModel() {
 
   var isSubmitButtonActive = ObservableField<Boolean>(false)
 
-  @Bindable
   fun getInputPinTextWatcher(): TextWatcher {
```

Comment:
Reminder comment: please verify that all of these text watchers still work for each of these places when testing manually in the app.

Reply:
Okay!